x/paramHub/hub.go

```diff
@@ -144,5 +154,10 @@ func init() {
 		"miniIssueMsg":                       fees.FixedFeeCalculatorGen,
 		"crossDistributeRewardRelayFee":      fees.FixedFeeCalculatorGen,
 		"crossDistributeUndelegatedRelayFee": fees.FixedFeeCalculatorGen,
+		"create_validator_open":              fees.FixedFeeCalculatorGen,
```

Comment:
Can we change this `create_validator_open` `edit_validator` to keep the same style?

Reply:
same here